.circleci/config.yml

```diff
@@ -140,8 +140,10 @@ jobs:
     executor: test-executor
     steps:
       - browser-tools/install-browser-tools:
-          chrome-version: 116.0.5845.96 # TODO: remove when chromedriver downloads are fixed
-          replace-existing-chrome: true
+          replace-existing-chrome: false
```

Comment:
question - do we even need to have `replace-existing-chrome` if it defaults to [false](https://circleci.com/developer/orbs/orb/circleci/browser-tools)?

Reply:
No  we do not, will remove